src/apify_client/clients/resource_clients/task.py

```diff
@@ -0,0 +1,214 @@
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/run-collection/run-task
+
+        Args:
+            task_input (dict, optional): Task input dictionary
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
+                                   By default, the run uses the build specified in the task settings (typically latest).
+            memory_mb (int, optional): Memory limit for the run, in megabytes. By default, the run uses a memory limit specified in the task settings.
+            timeout_secs: (int, optional): Optional timeout for the run, in seconds. By default, the run uses timeout specified in the task settings.
+            wait_for_finish: (bool, optional): The maximum number of seconds the server waits for the run to finish.
+                                               By default, it is 0, the maximum value is 300.
+            webhooks (list, optional): Specifies optional webhooks associated with the actor run, which can be used to receive a notification
+                                       e.g. when the actor finished or failed. Note: if you already have a webhook set up for the actor or task,
+                                       you do not have to add it again here.
+
+        Returns:
+            The run object
+        """
+        raise ApifyClientError('Method not yet finished. Run subclient needs to be implemented first.')
+
+        # run = self.start(
+        #     task_input=task_input,
+        #     build=build,
+        #     memory_mb=memory_mb,
+        #     timeout_secs=timeout_secs,
+        #     wait_for_finish=wait_for_finish,
+        #     webhooks=webhooks,
+        # )
+
+        # TODO - retrieve the run using Run client and wait on it
+
+    def get_input(self) -> Optional[Dict]:
+        """Retrieve the input for this task.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-input-object/get-task-input
+
+        Returns:
+            Retrieved task input
+        """
+        try:
+            response = self.http_client.call(
+                url=self._url('input'),
+                method='GET',
+                params=self._params(),
+            )
+            return cast(Dict, response.json())
+        except ApifyApiError as exc:
+            _catch_not_found_or_throw(exc)
+        return None
+
+    def update_input(self, *, task_input: Dict) -> Dict:
+        """Update the input for this task.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-input-object/update-task-input
+
+        Returns:
+            Retrieved task input
+        """
+        response = self.http_client.call(
+            url=self._url('input'),
+            method='PUT',
+            params=self._params(),
+            json=task_input,
+        )
+        return cast(Dict, response.json())
+
+    def last_run(self, *, status: Optional[ActorJobStatus] = None) -> Any:
```

Comment:
Maybe there could be an option to differentiate between the run that was started last and the run that finished last?

Reply:
Is there this functionality in JS client (@mnmkng )? I am not aware of it.

Reply:
The `lastRun` uses the [last run API endpoint](https://docs.apify.com/api/v2#/reference/actor-tasks/last-run-object-and-its-storages). There's no client magic behind it and I wouldn't add any at this point.

Reply:
Okey. It is just a little confusing what run is actually the last, so maybe just mention it in the function description?

Reply:
Good idea, I added it to the function description. Thx